cmd/root.go

```diff
@@ -100,7 +100,7 @@ type Command struct {
 	quiet                      bool
 
 	// impersonationChain is a comma separated list of one or more service
-	// accounts. The last entry in the chain is the impersonation target. Any
+	// accounts. The first entry in the chain is the impersonation target. Any
 	// additional service accounts before the target are delegates. The
```

Comment:
Is this supposed to say "after" instead of begin? That is what the Cloud SQL change had.
```suggestion
	// additional service accounts after the target are delegates. The
```